CMakeLists.txt

```diff
@@ -107,7 +106,6 @@ if (${CMAKE_BUILD_TYPE} STREQUAL DEBUG)
 else (${CMAKE_BUILD_TYPE} STREQUAL DEBUG)
 
     MESSAGE("cmake: RELEASE compilation")
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
```

Comment:
Seems to be ok:

$ VERBOSE=1 make release
...
cd /home/fermin/src/fiware-orion/BUILD_RELEASE/src/lib/common && /usr/bin/c++   -DLINUX -DLM_ON -DLONG_RAPIDXML_NAME_SPACE -Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -fno-var-tracking-assignments -DLINUX -O3 -DNDEBUG -I/home/fermin/src/fiware-orion/src/lib    -fPIC -o CMakeFiles/common.dir/statistics.cpp.o -c /home/fermin/src/fiware-orion/src/lib/common/statistics.cpp

Previous to this removal with have "...-DLINUX -O2 -O3...".